packages/cashier/src/pages/payment-agent/payment-agent-list/payment-agent-list.tsx

```diff
@@ -1,15 +1,38 @@
-import PropTypes from 'prop-types';
 import React from 'react';
-import { Loading, Tabs, Text } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
+import { Loading, Tabs, Text } from '@deriv/components';
 import { isDesktop, isMobile, website_name } from '@deriv/shared';
 import { connect } from 'Stores/connect';
+import { RootStore } from 'Types';
 import VerificationEmail from 'Components/verification-email';
 import PaymentAgentDeposit from '../payment-agent-deposit';
 import PaymentAgentWithdrawForm from '../payment-agent-withdraw-form';
 import PaymentAgentWithdrawalLocked from '../payment-agent-withdrawal-locked';
 import './payment-agent-list.scss';
 
+declare module 'react' {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    interface HTMLAttributes<T> {
+        label?: string;
+    }
+}
```

Comment:
@hamid-deriv Let's not do this 🙏🏻😬 

Can’t we just pass a `label` element as a child to the `div` instead?